routes/studentRoutes.js

```diff
@@ -4,7 +4,15 @@ var Student = require(path.join(__dirname, '../models/student'));
 
 routes.GETallStudents = function(req, res) {
   Student.find({}, function(err, allStudents) {
```

Comment:
Ok so I think we tried this, but the problem was that we needed to be able to pass back the students of a specific cohort to get their entries for the cohort, and the requests were getting really annoying.  If you have a better solution you can implement it if you want, but this works and it seemed like a lesser janky way of doing it then any thing else we could come up with.  :)